src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -17,14 +17,15 @@ public HelpCommand() {}
     public CommandResult execute() {
         return new CommandResult(
                 AddCommand.MESSAGE_USAGE
-                + "\n" + DeleteCommand.MESSAGE_USAGE
-                + "\n" + ClearCommand.MESSAGE_USAGE
-                + "\n" + FindCommand.MESSAGE_USAGE
-                + "\n" + ListCommand.MESSAGE_USAGE
-                + "\n" + ViewCommand.MESSAGE_USAGE
-                + "\n" + ViewAllCommand.MESSAGE_USAGE
-                + "\n" + HelpCommand.MESSAGE_USAGE
-                + "\n" + ExitCommand.MESSAGE_USAGE
+                        + "\n" + DeleteCommand.MESSAGE_USAGE
```

Comment:
This is good, make sure to update your `help` command as well, so that internally it also reflects the new commands to the user. But there seems to be an unnecessary repositioning of the code, try not to commit this, as it does not show clearly what the changes are. (adding sort command usage into help command)